src/Core/Framework/Bundle.php

```diff
@@ -158,4 +162,37 @@ protected function registerContainerFile(ContainerBuilder $container): void
         $loader = new XmlFileLoader($container, new FileLocator($this->getPath()));
         $loader->load($containerFilePath);
     }
+
+    protected function registerLogger(ContainerBuilder $container): void
+    {
+        $servicePrefix = $this->getContainerPrefix();
```

Comment:
Maybe an early check is already a logger definition in container? If someone wants to implement own thing

Reply:
Good one